scripts/o3de/o3de/repo_properties.py

```diff
@@ -484,6 +525,9 @@ def add_parser_args(parser):
     modify_object_group.add_argument('--download-prefix', '-dp', type=str, required=False,
                                    help='A URL prefix for a file attached to a GitHub release might look like this:'
                                         '-dp https://github.com/o3de/o3de-extras/releases/download/2305.0/')
+    modify_object_group.add_argument('--upload-git-release-tag', '-ugrt', type=str, default=False,
+                                   help='Automatically uploads your object-release-archive.zip file to specified github release.\n'
```

Comment:
Mention requiring read/write credentials

Reply:
It will be prompted when user calls the --upload-git-release-tag function